product_combination_exclude/models/product_attribute_exclude.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Graeme Gellatly
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ProductAttributeExclude(models.Model):
+
+    _name = 'product.attribute.exclude'
+    _description = 'Product Attribute Exclude'
+    _rec_name = 'attribute_value_ids'
+
+    attribute_value_ids = fields.Many2many(
+        comodel_name='product.attribute.value',
+        string='Incompatible Attributes'
```

Comment:
@gdgellatly it's not your idiom, but a Python+GIT best practice for avoiding 2 lines diff if you add any other argument that doesn't fit in this line.